common/constants.ts

```diff
@@ -150,6 +150,19 @@ export const GCP_ALLOW_UNAUTHENTICATED: string = readEnv(
   "true",
 );
 
+// Telemetry
+export const LOG_LEVEL: string = readEnv(
```

Comment:
`readEnv` now needs to take a list of possible value locations so we can check both `OTEL_LOG_LEVEL` and `LOG_LEVEL`